packages/web/app/src/components/ui/date-range-picker.tsx

```diff
@@ -231,6 +231,49 @@ export function DateRangePicker(props: DateRangePickerProps): JSX.Element {
     );
   };
 
+  // Dynamic presets
```

Comment:
useless comment? 😆

Reply:
TIP: It can be something like this: 
// The following code is suggesting the users filter options based on the input, when the input starts with "last" keyword.